nvtabular/inference/triton/ensemble.py

```diff
@@ -348,11 +351,20 @@ def generate_nvtabular_model(
     conts=None,
     max_batch_size=None,
     output_info=None,
+    sparse_max=None,
     backend="python",
 ):
-    """converts a workflow to a triton mode"""
+    """converts a workflow to a triton mode
+    Parameters
+    ----------
+    sparse_max:
+    Max length of the each row when the sparse data is converted to dense
+    """
 
     workflow.save(os.path.join(output_path, str(version), "workflow"))
+    if sparse_max:
+        with open(os.path.join(output_path, str(version), "workflow", "model_info.json"), "w") as o:
+            json.dump(sparse_max, o)
```

Comment:
Would it be possible to store this in the existing Triton model config file instead of creating a new json file? You can add custom information to the model config by adding a section like:
```
parameters {
  key: "sparse_max"
  value {
    int_value: 20
  }
}
```

Reply:
Good idea.